src/js/services/analytics.test.js

```diff
@@ -1,7 +1,6 @@
 import AnalyticsClient from './analytics.js';
 import { default as CookieOrginal } from 'js-cookie';
 import glob from '../../../jest/global';
-import { describe } from 'jest-circus';
```

Comment:
this is probably automatically imported by my VS Code 😬

Reply:
that no original web from CircleCI became i'm develop that